libs/template/create-oryx/package.json

```diff
@@ -1,13 +1,14 @@
 {
   "name": "create-oryx",
-  "version": "0.21.0",
+  "version": "0.33.0",
```

Comment:
Wow... 12 versions. Maybe it make sense to use rc for testing? :)

Reply:
Not 12, as new one started with `0.30`.
It was not about testing, as testing was done with verdaccio, but troubleshoting deployment issues.
Not sure if there is a point for rc either, as nothing/noone is relying on this version.